internal/node/node.bzl

```diff
@@ -283,8 +291,9 @@ The set of default  environment variables is:
 
 - `VERBOSE_LOGS`: use by some rules & tools to turn on debug output in their logs
 - `NODE_DEBUG`: used by node.js itself to print more logs
+- `RUNFILES_LIB_DEBUG`: print diagnostic message from Bazel runfiles.bash helper
```

Comment:
why not use `VERBOSE_LOGS` for that?

Reply:
`RUNFILES_LIB_DEBUG` is what is in the @bazel_tools runfiles.bash helper

Reply:
However, since we now vendor it in we could change that as well but I wanted to keep the changes to that file to minimum. I'll go back and comment around the rules_nodejs custom parts which add support for $(rootpath)